FundacionParaguaya/MentorApp
# Navigation issues

This fixes a series of navigation related issues in the life map process

**To Test**
1. Create a new life map. Go to family participan screen for the first time.
2. Tap on the X at the top right. A modal should appear that the draft will be deleted.
3. Tap No, the modal should close.
4. Open it again and click yes. You should be navigated to Dashboard and the draft deleted.
5. Create a new draft. Go through the screens to family member names.
6. Close the draft. It should be visible in the dashboard.
7. Go to family participant again and close it. The draft should be visible in the dashboard.
8. In the Dashboard open the navigation. The selected item should be Dashboard.
9. Open Create a life map, then the navigation. The selected item should correspond.
10. Create a new survey and go to family participant.
11. Click the back icon. A modal should appear. Clicking yes should go back and delete the draft.

**Related issues**
- Resolves #154
- Resolves #146
